src/applications/mhv/secure-messaging/components/Search/FilterBox.jsx

```diff
@@ -0,0 +1,230 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  VaDate,
+  VaModal,
+  VaSelect,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { useHistory } from 'react-router-dom';
+import moment from 'moment';
+import {
+  DateRangeValues,
+  DateRangeOptions,
+  SelectCategories,
+} from '../../util/inputContants';
+import { runAdvancedSearch } from '../../actions/search';
+import { dateFormat } from '../../util/helpers';
+
+const FilterBox = props => {
+  const {
+    folders,
+    testingSubmit,
+    testingDateRange,
+    testingFromDate,
+    testingToDate,
```

Comment:
Are these props used just for the unit testing?

Reply:
i went ahead and deleted these props from the filterbox component, and the test cases.